v2/devices/mimir.yml

```diff
@@ -0,0 +1,280 @@
+      - actions:
+          - fastboot:reboot_recovery:
+        fallback:
+          - core:user_action:
+              action: "recovery"
+        condition:
+          var: "bootstrap"
+          value: true
+      - actions:
+          - adb:reboot:
+              to_state: "recovery"
+        fallback:
+          - core:user_action:
+              action: "recovery"
+        condition:
+          var: "bootstrap"
+          value: false
+      - actions:
+          - systemimage:install:
+      - actions:
+          - adb:reboot:
+              to_state: "recovery"
+        fallback:
+          - core:user_action:
+              action: "recovery"
+    slideshow: []
+  - name: "Volla OS"
+    compatible_installer: ">=0.9.2-beta"
+    prerequisites: []
+    steps:
+      - actions:
+          - core:download:
+              group: "firmware"
+              files:
+                - url: "https://volla.tech/filedump/vollaos/volla-14.0-20250130-stable-BOOTSTRAP-mimir.zip"
+                  checksum:
+                    sum: "73891d2c7dd620dea44f7f3ad9ad5474cded5262470319f42349b1ee72ed1bc1"
+                    algorithm: "sha256"
+          - core:unpack:
+              group: "firmware"
+              files:
+                - archive: "volla-14.0-20250130-stable-BOOTSTRAP-mimir.zip"
+                  dir: "unpacked"
+      - actions:
+          - adb:reboot:
+              to_state: "bootloader"
+        fallback:
+          - core:user_action:
+              action: "bootloader"
+      - actions:
+          - fastboot:set_active:
+              slot: "a"
+          - fastboot:flash:
+              partitions:
+                - partition: "boot_a"
+                  file: "unpacked/boot.img"
+                  group: "firmware"
+                - partition: "csci"
+                  file: "unpacked/csci.ini"
+                  group: "firmware"
+                - partition: "dpm_a"
+                  file: "unpacked/dpm.img"
+                  group: "firmware"
+                - partition: "dtbo_a"
+                  file: "unpacked/dtbo.img"
+                  group: "firmware"
+                - partition: "gz_a"
+                  file: "unpacked/gz.img"
+                  group: "firmware"
+                - partition: "lk_a"
+                  file: "unpacked/lk.img"
+                  group: "firmware"
+                - partition: "logo"
+                  file: "unpacked/logo.bin"
+                  group: "firmware"
+                - partition: "mcupm_a"
+                  file: "unpacked/mcupm.img"
+                  group: "firmware"
+                - partition: "md1img_a"
+                  file: "unpacked/md1img.img"
+                  group: "firmware"
+                - partition: "pi_img_a"
+                  file: "unpacked/pi_img.img"
+                  group: "firmware"
+                - partition: "preloader_a"
+                  file: "unpacked/preloader_tb8781p1_64.bin"
```

Comment:
Can this be flashed safely by `fastboot`? (just having some flashbacks about different header images for preloader on previous devices).

Reply:
Disconnect in the middle of that or similar would throw device to BROM mode on boot with blank screen until fixed and would have to be recovered via e.g. `mtk wf --preloader preloader_tb8781p1_64.bin --parttype boot1 preloader_tb8781p1_64.bin` using https://github.com/bkerler/mtkclient

It can likely be omitted too if preferred. fwiw all other Volla yamls also have it atm. similarly though even `lk` or even other partitions being partially flashed could cause problems but those most likely will need changing when jumping between android versions anyway

Reply:
Probably let's keep it, as otherwise we will need to get this back eventually once Volla OS gets updated to a newer BSP.